sympy/algebras/tests/test_quaternion.py

```diff
@@ -136,3 +146,15 @@ def test_quaternion_rotation_iss1593():
                 [1,      0,      0],
                 [0, cos(x), -sin(x)],
                 [0, sin(x), cos(x)]]))
+
+
+def test_quaternion_nultiplication():
+    q1 = Quaternion(3 + 4*I, 2 + 5*I, 0, 7 + 8*I, real_field = False)
```

Comment:
What do these quaternions with complex coefficients represent? Is this biquaternions of some type?
https://en.wikipedia.org/wiki/Biquaternion

Reply:
No idea really. Just copied from another test. I would assume that the complex I is not the same as the quaternion I, so just complex values of each "dimension".